### ISSUE TITLE
v1 update?

### ISSUE THREAD

**Issue by omni72** [gratrix/vcv-gratrix#12]
Hi Sam - will your Gratrix modules be finding their way into the new v1 plugin library any time soon? :) 

Take care -

Omni

**Comment by polyzium** [comment]
Because Gratrix focuses on polyphony, and polyphonic cables have been added in Rack v1, I don't think it'll make its way to v1 plugin library. It made this module pack obsolete.

**Comment by earthvomit** [comment]
I don't agree with you that they are obsolete. At least not Binary. It used to be in most of my pre V1 patches.Could you please do it. If not I want to try although I have no programming experience.

**Comment by spock171** [comment]
Hello,

I migrated some of the modules to VCV v1.

Greetings Frank.

**Comment by earthvomit** [comment]
I don't see it . All I see is 3 year old Gratrix on github. No gratrix in plugin manager.
Greetings Finnur.

**Comment by spock171** [comment]
You are right, the v1-Version are not yet published. I did it outside GitHub, only for private, and want to confirm with Sam to publish it.
Greetings frank

**Comment by earthvomit** [comment]
Nice. Thank you so much. I am so exited as I have tons of patches with gratrix-binary.

**Comment by spock171** [comment]
The following modules are currently updated to VCV v1:
ADSR-F1, Binary-G1, the Blanks, Chord-G1, Fade-G1, Fade-G2, Keys-G1, Octave-G1, Seq-G1, Seq-G1, Scope-G1, VCA-F1, VCO-F1, VCO-F2, VU-G1.

I didn't change the graphics.
I think, On Monday I will publish the package for Windows here. for testing and more ideas.

**Comment by spock171** [comment]
Here you can find the code and Windows-Build:
https://github.com/spock171/1-Seifert
